Minor updates to access review beta to expose top-level decisions navigation route (#15036)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accessreviewinstancedecisionitemazureroleresource.md

---
title: "accessReviewInstanceDecisionItemAzureRoleResource resource type"
description: "Every decision item in an access review represents a principal's access to a resource. accessReviewInstanceDecisionItemAzureRoleResource represents that the decision item represents an azure role resource."
author: "isabelleatmsft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# accessReviewInstanceDecisionItemAzureRoleResource resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]

> Every decision item in an access review represents a principal's access to a resource. All accessReviewInstanceDecisionItemAzureRoleResource objects represent access to Azure resource roles. accessReviewInstanceDecisionItemAzureRoleResource is an open type that allows for other properties to be passed in.

Reviewer: ```suggestion
Represents access to Azure resource roles. accessReviewInstanceDecisionItemAzureRoleResource is an open type that allows for other properties to be passed in.

Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).

```

> Every decision item in an access review represents a principal's access to a resource. All accessReviewInstanceDecisionItemAzureRoleResource objects represent access to Azure resource roles. accessReviewInstanceDecisionItemAzureRoleResource is an open type that allows for other properties to be passed in.

Reviewer: ```suggestion
Represents access to Azure resource roles. accessReviewInstanceDecisionItemAzureRoleResource is an open type that allows for other properties to be passed in.

Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).

```

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |displayName|String|Display name of the resource|
> |id|String|Resource ID|
> |type|String|Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.|
> |scope|[accessReviewInstanceDecisionItemResource](../resources/accessreviewinstancedecisionitemresource.md)| Details of the scope this role is associated with|

Reviewer: Same changes as before apply here
```suggestion
| displayName | String | Display name of the resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
| id | String | Identifier of the decision item resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
| type | String | Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
| scope | [accessReviewInstanceDecisionItemResource](../resources/accessreviewinstancedecisionitemresource.md) | Details of the scope this role is associated with. |
```

## Relationships
None.

## JSON representation
The following is a JSON representation of the resource.
<!-- {

>   "blockType": "resource",
>   "@odata.type": "microsoft.graph.accessReviewInstanceDecisionItemAzureRoleResource",
>   "openType": true

Reviewer: ```suggestion
  "blockType": "resource",
  "@odata.type": "microsoft.graph.accessReviewInstanceDecisionItemAzureRoleResource",
  "baseType": "microsoft.graph.accessReviewInstanceDecisionItemResource",
  "openType": true
```

---
title: "accessReviewInstanceDecisionItemAzureRoleResource resource type"
description: "Represents access to Azure resource roles."
author: "isabelleatmsft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# accessReviewInstanceDecisionItemAzureRoleResource resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]

> Represents access to Azure resource roles. accessReviewInstanceDecisionItemAzureRoleResource is an open type that allows for other properties to be passed in.

Reviewer: ```suggestion
Represents Azure resource roles to which access has been granted through an [accessReviewInstanceDecisionItem]((https://docs.microsoft.com/en-us/graph/api/resources/accessreviewinstancedecisionitem?view=graph-rest-beta) object. **accessReviewInstanceDecisionItemAzureRoleResource** is an open type that allows for other properties to be passed in.
```

Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).

## Properties
|Property|Type|Description|
|:---|:---|:---|

> | displayName | String | Display name of the resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|

Reviewer 2: ```suggestion
| displayName | String | Display name of the Azure role. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
```

| id | String | Identifier of the decision item resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |

> | type | String | Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |

Reviewer 2: ```suggestion
| type | String | Type of resource. Type will always be `AzureRole`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
```
